cypress/e2e/27-cloud.cy.ts

```diff
@@ -19,100 +19,72 @@ describe('Cloud', { disableAutoLogin: true }, () => {
 		planData.expirationDate = fiveDaysFromNow.toJSON();
 	});
 
-	describe('BannerStack', () => {
-		it('should render trial banner for opt-in cloud user', () => {
-			cy.intercept('GET', '/rest/admin/cloud-plan', {
-				body: planData,
-			}).as('getPlanData');
-
-			cy.intercept('GET', '/rest/settings', (req) => {
-				req.on('response', (res) => {
-					res.send({
-						data: { ...res.body.data, deployment: { type: 'cloud' }, n8nMetadata: { userId: 1 } },
-					});
+	beforeEach(() => {
+		cy.intercept('GET', '/rest/admin/cloud-plan', {
+			body: planData,
+		}).as('getPlanData');
+
+		cy.intercept('GET', '/rest/settings', (req) => {
+			req.on('response', (res) => {
+				res.send({
+					data: {
+						...res.body.data,
+						deployment: { type: 'cloud' },
+						n8nMetadata: { userId: 1 },
+					},
 				});
-			}).as('loadSettings');
-
-			cy.signin({ email: INSTANCE_OWNER.email, password: INSTANCE_OWNER.password });
-
-			cy.visit(workflowPage.url);
-
-			cy.wait('@getPlanData');
+			});
+		}).as('loadSettings');
 
-			bannerStack.getters.banner().should('be.visible');
-
-			mainSidebar.actions.signout();
+		cy.intercept('GET', new RegExp('/rest/projects*')).as('projects');
+		cy.intercept('GET', new RegExp('/rest/roles')).as('roles');
+	});
 
-			bannerStack.getters.banner().should('not.be.visible');
+	function visitWorkflowPage() {
+		cy.visit(workflowPage.url);
+		cy.wait('@getPlanData');
+		cy.wait('@projects');
+		cy.wait('@roles');
+	}
 
+	describe('BannerStack', () => {
+		it('should render trial banner for opt-in cloud user', () => {
 			cy.signin({ email: INSTANCE_OWNER.email, password: INSTANCE_OWNER.password });
 
-			cy.visit(workflowPage.url);
+			visitWorkflowPage();
 
 			bannerStack.getters.banner().should('be.visible');
 
 			mainSidebar.actions.signout();
-		});
 
-		it('should not render opt-in-trial banner for non cloud deployment', () => {
```

Comment:
removed test. Not a very useful test anyways because that's the default in every other case.